source/_integrations/apsystems.markdown

```diff
@@ -3,22 +3,27 @@ title: APsystems
 description: Control and monitor your APsystems EZ1 microinverters locally without the cloud
 ha_iot_class: Local Polling
 ha_config_flow: true
-ha_release: 2024.6
+ha_release: 2024.7
```

Comment:
This integration is released in 2024.6 right?

Reply:
No, afaik it will be released tomorrow.

Reply:
Yup, 2024.6 is being released tomorrow...

Reply:
Oh, it's not my day today 🤦🏼 , you're right. Will fix that now.